RecoTracker/FinalTrackSelectors/src/TrackMVAClassifierBase.cc

```diff
@@ -41,18 +43,19 @@ TrackMVAClassifierBase::TrackMVAClassifierBase( const edm::ParameterSet & cfg )
 
 }
 
+
+#include "DataFormats/VertexReco/interface/Vertex.h"
```

Comment:
Is there any reason for having this #include here, and not on top together with the other ones?